src/Tempest/Http/src/Commands/MakeControllerCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Commands;
+
+use Tempest\Console\ConsoleArgument;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Generation\DataObjects\StubFile;
+use Tempest\Generation\Exceptions\FileGenerationAbortedException;
+use Tempest\Generation\Exceptions\FileGenerationFailedException;
+use Tempest\Generation\HasGeneratorConsoleInteractions;
+use Tempest\Http\Stubs\ControllerStub;
+
+final class MakeControllerCommand
+{
+    use HasGeneratorConsoleInteractions;
+
+    #[ConsoleCommand(
+        name: 'make:controller',
+        description: 'Creates a new controller class with a route',
+        aliases: ['controller:make', 'controller:create', 'create:controller'],
+    )]
+    public function __invoke(
+        #[ConsoleArgument(
+            help: 'The name of the controller class to create',
+        )]
+        string $className,
+        #[ConsoleArgument(
+            help: 'The path of the route',
+        )]
+        ?string $controllerPath = null,
+        #[ConsoleArgument(
+            help: 'The name of the view returned from the controller',
+        )]
+        ?string $controllerView = null,
+    ): void {
+        $suggestedPath = $this->getSuggestedPath($className);
+        $targetPath = $this->promptTargetPath($suggestedPath);
+        $shouldOverride = $this->askForOverride($targetPath);
```

Comment:
Is there a reason we specifically _ask_ for the override? Why not a flag that's false by default? `bool $override = false`.

Reply:
I've done this to avoid another parameter to add on each individual make command.
This also trigger only when the file exists.
If I'm not wrong, the `--force` is global and can skip all confirm interactions right ?

Reply:
> If I'm not wrong, the --force is global and can skip all confirm interactions right ?

Only if you add the `ForceMiddleware`. Let's keep it for now though, I'll give it some more thought after this PR has been merged :)